### PR TITLE
Fix issue with ephemeral reference count

### PR DESCRIPTION
**Pull request by corrideat**

### CODE REVIEW

File: frontend/model/contracts/shared/functions.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ export function leaveChatRoom (contractID: string) {
     sbp('state/vuex/commit', 'setCurrentChatRoomId', { groupID: rootState.currentGroupId })
   }
 
-  sbp('gi.actions/identity/kv/deleteChatRoomUnreadMessages', { contractID })
+  await sbp('gi.actions/identity/kv/deleteChatRoomUnreadMessages', { contractID })
```
Comment (Silver-IT) [inline frontend/model/contracts/shared/functions.js]: Why do we need to wait this KV action? It's to save unreadMessages in identityContractID space, not chatRoomID space.

**Review by Silver-IT — CHANGES_REQUESTED** [review]
Nice work, @corrideat!

I can see the changes of this PR is to wait `gi.actions/identity/kv/deleteChatRoomUnreadMessages`, and I made one question.

File: frontend/model/contracts/shared/functions.js (hunk 1 above)
Comment (corrideat) [reply frontend/model/contracts/shared/functions.js]: In that case, you're right that waiting shouldn't be needed.

**Review by Silver-IT — DISMISSED** [review]
Nice work, @corrideat!

**Review by taoeffect — APPROVED** [review]
Probably OK

**Pull request merged**